src/background/services/secrets/models.ts

```diff
@@ -26,6 +27,28 @@ export enum SecretType {
   Fireblocks = 'fireblocks',
 }
 
+export type Secp256k1 = 'secp256k1';
+export type Ed25519 = 'ed25519';
+export type Curve = Secp256k1 | Ed25519;
+export const EVM_BASE_DERIVATION_PATH = "m/44'/60'/0'";
+export const AVALANCHE_BASE_DERIVATION_PATH = "m/44'/9000'/0'";
```

Comment:
These could probably use a better name (those are paths for the legacy `xpub` and `xpubXP`).